cmd/utils/dev.go

```diff
@@ -276,3 +279,10 @@ func GetApp(ctx context.Context, dev *model.Dev, c kubernetes.Interface) (apps.A
 	}
 	return app, false, nil
 }
+
+func doesAutocreateAppExist(ctx context.Context, dev *model.Dev, c kubernetes.Interface) bool {
+	autocreateDev := *dev
+	autocreateDev.Name = model.DevCloneName(dev.Name)
+	_, err := apps.Get(ctx, &autocreateDev, dev.Namespace, c)
+	return err == nil
```

Comment:
This is a general assumption I see we are doing in several places, even in the backend. The error could be related to any Kubernetes API error o timeout. A temporal error in K8s could make the tests fail. Should we retry if the error != nil and it isn't a not found error? I think that would make a bit more fault-tolerant the test